src/main/java/uk/gov/hmcts/reform/idam/service/DeleteUserService.java

```diff
@@ -50,7 +50,11 @@ public void deleteUser(String userId) {
                 response.status()
             );
             log.error(msg);
-            throw new IdamApiException(msg);
+            failedDeletions++;
```

Comment:
We need to modify the catch statement too, if idamClient throw an  exception, that exception will be caught and propagated up the call stack to the calling methods.  The calling methods have no exception handling in place and therefore we will end up in the scenario we want to fix. We also need to update failedDeletions in the catch statement too.